src/ott/datasets.py

```diff
@@ -152,3 +319,52 @@ def create_gaussian_mixture_samplers(
   )
   dim_data = 2
   return train_dataset, valid_dataset, dim_data
+
+
+def create_sklearn_samplers(
+    source_kwargs: Mapping[str, Any] = MappingProxyType({}),
+    target_kwargs: Mapping[str, Any] = MappingProxyType({}),
+    train_batch_size: int = 256,
```

Comment:
Please change the signature to
```
trn_batch_size: int = 256,
val_batch_size: int = 256,
source_kwargs: ...
target_kwargs: ...
```

Reply:
Maybe consider removing the default batch size? Don't have any preference, will leave it up to you!